health-services/libraries/health-services-common/CHANGELOG.md

```
@@ -1,5 +1,9 @@
 All notable changes to this module will be documented in this file.
 
+## 1.0.22 - 2025-06-04
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)